docs/en_US/Blog/ParallelizingTpeSearch.md

```diff
@@ -0,0 +1,101 @@
+# Parallelizing a Sequential Algorithm TPE
+
```

Comment:
Seems the folder name has a conflict with the master now.  Suggest syncing the latest master.

Reply:
Will handle this after @Crysple finish the experiment